pyswarms/base/base_single.py

```diff
@@ -224,7 +224,7 @@ def _populate_history(self, hist):
         self.pos_history.append(hist.position)
         self.velocity_history.append(hist.velocity)
 
-    def optimize(self, objective_func, iters, print_step=1, verbose=1):
+    def optimize(self, objective_func, iters, args=(), kwargs={}, print_step=1, verbose=1):
```

Comment:
Maybe `*args` and `**kwargs` will work here instead of `args=()` and `kwargs={}`?